### PR TITLE
Allow passworded servers

### PR DESCRIPTION
**Pull request by lucasgautheron**
#89 
Feel free to close this if unwanted.

### CODE REVIEW

**Comment by lucasgautheron** [comment]
Next step is to add this to live games/pinger service.

File: reference-reader/src/main/scala/af/rr/ServerRecord.scala [hunk 1]
```diff
@@ -22,11 +22,13 @@ object ServerRecord {
         hostname <- Option(rec.get(1)).filter(_.nonEmpty)
         port <- Try(rec.get(2).toInt).toOption
         kind <- Option(rec.get(3)).filter(_.nonEmpty)
+        password <- Option(rec.get(4)).filter(_.nonEmpty)
```
Comment (ScalaWilliam) [inline reference-reader/src/main/scala/af/rr/ServerRecord.scala]: The tests tell you that this is wrong. In cmd, while developing, do:
"sbt ~reference-reader/test"

**Comment by lucasgautheron** [comment]
Might as well use this as an opportunity to remove this https://github.com/ScalaWilliam/ActionFPS/blob/6f966f44650e8decb9ecd78c10af8f0aabe7c49c/pinger-client/src/main/scala/acleague/pinger/package.scala#L34 and use the spreadsheet instead

**Comment by ScalaWilliam** [comment]
That'd be a separate ticket

**Comment by ScalaWilliam** [comment]
Lgtm.

**Comment by ScalaWilliam** [comment]
Did you test it locally?

**Comment by lucasgautheron** [comment]
All I did was performing the existing tests ATM. Since you want to treat the server page first alone then i'll test it asap

**Comment by ScalaWilliam** [comment]
Well sanity tests are always useful, ie start up locally, see if what comes out the other end is good. Btw the code seems fine otherwise.

**Comment by ScalaWilliam** [comment]
* Checks. Don't do it if you think unnecessary

**Comment by ScalaWilliam** [comment]
:+1:

**Pull request merged**